hack/util.sh

```diff
@@ -351,21 +356,21 @@ function wait_for_url {
 	wait=${3:-0.2}
 	times=${4:-10}
 
-	set_curl_args $wait $times
+	set_curl_args "${wait}" "${times}"
 
 	set +e
 	cmd="env -i CURL_CA_BUNDLE=${CURL_CA_BUNDLE:-} $(which curl) ${clientcert_args} -fs ${url}"
-	for i in $(seq 1 $times); do
+	for i in $(seq 1 "${times}"); do
 		out=$(${cmd})
 		if [ $? -eq 0 ]; then
 			set -e
 			echo "${prefix}${out}"
 			return 0
 		fi
-		sleep $wait
+		sleep "${wait}"
 	done
 	echo "ERROR: gave up waiting for ${url}"
-	echo $(${cmd})
+	${cmd}
```

Comment:
You will be missing a blank line, if that's what's expected, in cases where `${cmd}` prints no output.  It's probably OK though.